text-notification/index.js

```diff
@@ -0,0 +1,85 @@
+            action: this.onMenuPress.bind(this)
+        })
+
+        this.menus.register({
+            id: 'notify-config',
+            section: 'plugin-settings',
+            panel: {
+                fields: [
+                    { type: 'section', name: 'Phone Numbers' },
+                    { type: 'text', id: 'phone-number1', name: 'Phone Number', help: 'Enter the phone number you wish to be alerted when a user joins the space. Example Format: +15555555555' },
+                    { type: 'text', id: 'phone-number2', name: 'Phone Number', help: 'Enter the phone number you wish to be alerted when a user joins the space. Example Format: +15555555555' },
+                    { type: 'text', id: 'phone-number3', name: 'Phone Number', help: 'Enter the phone number you wish to be alerted when a user joins the space. Example Format: +15555555555' },
+                    { type: 'text', id: 'phone-number4', name: 'Phone Number', help: 'Enter the phone number you wish to be alerted when a user joins the space. Example Format: +15555555555' },
+                    { type: 'text', id: 'phone-number5', name: 'Phone Number', help: 'Enter the phone number you wish to be alerted when a user joins the space. Example Format: +15555555555' },
+                ]
+            }
+        })
+
+    }
+    /**
+     * Updates the plugin when settings have changed
+     * @param {} feild that has been updated
+     * @param {any} value new value of the feild
+     */
+    onSettingsUpdated(feild, value){
+        //check that the number is of the correct format then add to set
+        if(value.match(/^\+[1-9]\d{1,14}$/)){
+            this.phoneNumberSet.add(value)
+        }
+        else{
+            this.menus.alert('Please renter the number with the correct format eg. +11234567890', 'Invalid international number', 'info')
+        }
+    }
+
+    /** Called when the user presses the Notify button */
+    async onMenuPress() {
+        // Ask user for message
+        if(this.phoneNumberSet.size > 0){
+            this.phoneNumberSet.forEach( (number) => {
+                let dataToSend = this.userName + ':' + number       //message body format: "userName:number"
+                fetch('https://us-central1-ydangle-high-fidelity-test-2.cloudfunctions.net/sendSMSNotification',{
+                    method: 'POST',
+                    mode: 'no-cors',
+                    headers: {
+                        'Content-Type': 'text/plain'
+                    },
+                    body: dataToSend,
+                })
+                .then(() =>{
+                    console.log('success message sent')
+                })
+                .catch(error => console.log('ERROR', error))
+            })
+            this.menus.alert('The hosts have been notified of your arrival','Notification sent', 'info')
```

Comment:
The icon here should be `success` instead of `info`, as it indicates that the notification has been sent successfully